pkg/KubeArmorController/api/security.kubearmor.com/v1/common.go

```diff
@@ -297,7 +297,7 @@ type ActionType string
 type Syscall string
 
 type SyscallFromSourceType struct {
-	Path      MatchPathType `json:"path,omitempty"`
+	Path      MatchPathType `json:"path"`
```

Comment:
```suggestion
	Path      MatchPathType `json:"path,omitempty"`
```
Path is not mandatory in SysCallFromSource